ReactTask/src/views/PatientProfile.js

```diff
@@ -0,0 +1,254 @@
+import React, { useEffect, useState } from 'react';
+import {
+  StylBoxRowPadding,
+  StylBoxPatients,
+  StylBoxDoctor,
+  StylBoxFlexColumn,
+  StylBoxColumnFlex,
+  StylBoxPatientContent,
+  StylBoxBtnPatients,
+  StylBoxRow,
+  BtnPatientsHiddenMob,
+  BtnCreateAppointment,
+  BtnPatients,
+  StylSubTitlePosition,
+  StylTitlePatients,
+  StylTitleName,
+  AvatarActive,
+  CardProfilePatient,
+  BoxFilter,
+  StylTitlePatientAppointment,
+  StylTitleCurrentPatientAppointment,
+  StylBoxRowFlexStart,
+  StylChooseDateAndTimeVisitDoctor,
+  StylItemChooseDoctor,
+  StylTitleVisitDoctor,
+  StylInputVisitDoctor,
+  StylArrowVisitDoctor,
+  StylBoxMenuSelectVisitDoctor,
+  StylElementSelectVisitDoctor,
+  StylBoxColumnFlexMargin,
+  CalendarVisitDoctor,
+  StylBoxChooseTime,
+  StylBtnChooseTimeVisitDoctor,
+  StylBtnSubmitFormVisitDoctor,
+} from '../components/';
+
+import patient from '../img/avatars/avatar-patient.png';
+import patient1 from '../img/avatars/patients-13.png';
+import logo from '../img/logo.png';
+import data from './../db/dbProfilePatient.json';
+import {
+  nameDoctorsArr,
+  ocuppationDoctorArr,
+  possibleTimeArr,
+} from './../constants/index';
+
+const PatientProfile = ({ title }) => {
+  const dbData = [data],
```

Comment:
It's bad to read and understand for me

Reply:
@matthew1232002 any suggestions for naming?

Reply:
@codezavr I'm about code boilerplate

Reply:
ReBuild `PatientProfile`

Reply:
Resolved